packages/composables/package.json

```diff
@@ -17,12 +17,17 @@
 	"license": "GPL-3.0",
 	"author": "Rijk van Zanten <rijkvanzanten@me.com>",
 	"exports": {
-		".": "./dist/index.js",
+		".": {
+			"types": "./dist/index.d.ts",
+			"import": "./dist/index.js"
+		},
 		"./package.json": "./package.json"
 	},
 	"main": "dist/index.js",
+	"types": "dist/index.d.ts",
 	"files": [
-		"dist"
+		"dist",
+		"!**/*.d.ts.map"
```

Comment:
We don't generate declaration maps anymore, so we can omit this one.
However, the files we should actually exclude are the tests (`!**/*.test.{js,d.ts}`).

Reply:
I wasn't sure if there was a specific case where they would be generated. I'll remove it.

I thought about excluding tests as well, but then I wasn't sure if this should be included in this PR and if it is wanted at all. But I will update that.

Reply:
Awesome, thanks 🚀